src/taco-dependency-installer/installers/androidPackagesInstaller.ts

```diff
@@ -77,26 +82,25 @@ class AndroidPackagesInstaller extends InstallerBase {
         }
     }
 
-    private getAvailableAndroidPackages(): Q.Promise<any> {
-        var deferred: Q.Deferred<any> = Q.defer();
+    private executeAndroidCommand(name: string, args: string[]): IExecuteCommandResult {
```

Comment:
It was just in case we wanted the callers to let the execute command know whether there is going to be a prompt or not, but it's not really needed.

I think this is good as is :+1: